en/docs/tutorials/scenarios/scenario7-analytics.md

```diff
@@ -1,4 +1,50 @@
 # Scenario 7 - Analytics
 
-!!! Note
-    Content to be added.
\ No newline at end of file
+## User Story
+
+**GOGO Transit has identified a new business opportunity; since the train arrival data contain the number of passengers in each arrival, it could be used to predict the crowd in the station for a certain time period. Therefore, the services built around the station such as stores in mall and cab services could use this data to optimize the number of employees they require for the services for each day. Thus, the GOGO decides to provide the passenger details API to the public with the subscription capability that generates revenue based on the number of successful requests completed within a time period.**
```

Comment:
```suggestion
GOGO Transit has identified a new business opportunity; since the train arrival data contains the number of passengers in each arrival, it could be used to predict the crowd in the station for a certain time period. Therefore, the services built around the station such as stores in the mall and cab services could use this data to optimize the number of employees they require for the services for each day. Thus, GOGO Transit decides to provide the passenger details API to the public with the subscription capability that generates revenue based on the number of successful requests completed within a time period.
```

Reply:
Fixed